src/js/nizovi.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-unused-vars */
+// 1. zadatak
+const slova = ['s', 'a', 'e']
+// 2. zadatak
+const brojevi = [1, 2, 3]
+
+// 4. zadatak
+const niz = []
+const max = 10000000
+const min = 1000000
+
+for (let i = 0; i < 50; i++) {
+  niz[i] = Math.floor(Math.random() * (max - min + 1)) + min
```

Comment:
Super!
U zadatku je stajalo "svi brojevi treba da budu različiti", bilo bi dobro ako možeš i to da odradiš.